lenovo/legion/15ich/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, ... }:
```

Comment:
Can you also add this file to the README and the flake.nix as described in https://github.com/NixOS/nixos-hardware/blob/master/CONTRIBUTING.md#2-adding-entry ?

Reply:
Added to both.